mnefun/_mnefun.py

```diff
@@ -949,7 +949,6 @@ def run_sss_locally(p, subjects, run_indices):
             raw.fix_mag_coil_types()
             raw = filter_chpi(raw)
 
-            assert isinstance(p.trans_to, string_types)
             if p.trans_to is 'median':
```

Comment:
crap, not sure how this got in, string comparisons should be `==` not `is`.

Also, why remove this assertion?